src/raftstore/store/worker/region.rs

```diff
@@ -243,6 +248,7 @@ impl SnapContext {
         Ok(())
     }
 
+    /// Handles generated snapshot task of the Region.
```

Comment:
I don't know what does "generated snapshot task" mean.. 🤣 Could you explain it more?